gtsam/linear/GaussianBayesNet.cpp

```diff
@@ -243,5 +243,25 @@ namespace gtsam {
   }
 
   /* ************************************************************************* */
+  double GaussianBayesNet::logNormalizationConstant() const {
+    /*
```

Comment:
Is this copy/paste from gc ?

Reply:
Not quite. It collects all the `logNormalizationConstants` from the GCs.